airbyte-cdk/python/unit_tests/sources/mock_server_tests/test_mock_server_abstract_source.py

```diff
@@ -242,7 +242,7 @@ def test_full_refresh_sync(self, http_mocker):
         assert len(actual_messages.state_messages) == 1
         validate_message_order([Type.RECORD, Type.RECORD, Type.STATE], actual_messages.records_and_state_messages)
         assert actual_messages.state_messages[0].state.stream.stream_descriptor.name == "users"
-        assert actual_messages.state_messages[0].state.stream.stream_state == {"__ab_no_cursor_state_message": True}
+        assert actual_messages.state_messages[0].state.stream.stream_state == AirbyteStateBlob(__ab_no_cursor_state_message=True)
```

Comment:
This will affect some mock server tests like [this one](https://github.com/airbytehq/airbyte/blob/8e97ace3660fd764b35cfaec27866776416e10f0/airbyte-integrations/connectors/source-github/unit_tests/integration/test_events.py#L180-L182)